lib/SearchAndSort/SearchAndSort.js

```diff
@@ -206,12 +206,6 @@ class SearchAndSort extends React.Component {
     this.transitionToParams = this.transitionToParams.bind(this);
   }
 
-  componentDidMount() {
```

Comment:
Why where you able to straight-up remove this? Was it not doing anything for us?

Reply:
Yes exactly. It was not doing anything for us. The component would get mount before we had any resources available.

Reply:
Looks like you added it in https://github.com/folio-org/stripes-smart-components/commit/3a3c056a23019eba2a03d7927ea1d1ef07f89da8 :-)

Reply:
@MikeTaylor something weird is happening with github because I don't see it being added anymore
https://github.com/folio-org/stripes-smart-components/pull/233/files

Reply:
@MikeTaylor ah never mind. You pointed to a commit from April which I thought was my commit from today and got confused. Yes that commit was wrong.

Reply:
@MikeTaylor actually thank you for pointing that out. I will need to take a closer look now.